core/src/main/scala/kafka/server/AdminManager.scala

```diff
@@ -179,6 +179,9 @@ class AdminManager(val config: KafkaConfig,
         CreatePartitionsMetadata(topic.name, assignments, ApiError.NONE)
       } catch {
         // Log client errors at a lower level than unexpected exceptions
+        case e: TopicExistsException =>
+          debug(s"Topic '${topic.name}' already exists.", e)
```

Comment:
Should we make it clear that this is during topic creation?

Reply:
@ijuma Thanks for the review, updated.